### PR TITLE
Use environment variable to authenticate to Google Drive

### PR DESCRIPTION
**Pull request by antidipyramid**
## Overview

This PR transitions the app to use an environment variable to authenticate to Google Drive rather than the current JSON file.

Connects Metro-Records/la-metro-dashboard#123 #1043

## Testing Instructions

 * Decrypt `configs/lametro_service_acct_key.json` and copy the contents to `GOOGLE_SERVICE_ACCT_API_KEY` in `.env.local` (you'll have to move off this branch briefly to do + remove newline characters from the JSON string)
 * Set the `REMOTE_ANALYTICS_FOLDER` value in `.env.local` (you can use the value set in the staging app's Heroku env vars)
 * Run `python manage.py generate_tag_analytics`
 * Verify that the analytics file has been uploaded

### CODE REVIEW

File: councilmatic/settings.py [hunk 1]
```diff
@@ -174,6 +174,8 @@
 # - Analytics
 ANALYTICS_TRACKING_CODE = env("ANALYTICS_TRACKING_CODE")
 SERVICE_ACCOUNT_KEY_PATH = "configs/lametro_service_acct_key.json"
```
Comment (fgregg) [inline councilmatic/settings.py:176]: don't you want to remove this line?

**Review by fgregg — CHANGES_REQUESTED** [review]
should we also remove references to settings_deployment that are sprinkled about, or do you want to save that for a separate clean up PR?

**Review by fgregg — CHANGES_REQUESTED** [review]
we should be able to remove the keyrings directory altogether now.

**Comment by fgregg** [comment]
> should we also remove references to settings_deployment that are sprinkled about, or do you want to save that for a separate clean up PR?

no, this should be done in a separate PR

**Comment by antidipyramid** [comment]
@fgregg Any more comments on this one?

**Review by fgregg — APPROVED** [review]

**Pull request merged**